app/static/horoscope.js

```diff
@@ -0,0 +1,12 @@
+function daily_horoscope(sign) {
+    var x = 'https://aztro.sameerkumar.website/?sign=' + sign + '&day=today'
```

Comment:
Maybe use a more descriptive  naming?

Reply:
Prefer const and more descriptive naming